tests/AgDatabaseMove.Integration/TestAgDatabase.cs

```diff
@@ -25,5 +25,28 @@ public void TestLoginSidsMatch()
       var sid = _fixture._createdLogins.First().Sid;
       Assert.All(_fixture._createdLogins.Select(l => l.Sid), s => Assert.Equal(s, sid));
     }
+
+    [Fact]
+    public void TestUserExists()
+    {
+      var db = _fixture._agDatabase._listener.Primary.Database(_fixture._agConfig.DatabaseName);
+      Assert.Contains(_fixture._loginConfig.LoginName, db.Users.Select(u => u.Name));
+    }
+
+    [Fact]
+    public void TestUserHasRoles()
+    {
+      var db = _fixture._agDatabase._listener.Primary.Database(_fixture._agConfig.DatabaseName)._database;
+      var user = db.Users[_fixture._loginConfig.LoginName];
+      Assert.True(user.EnumRoles().Contains("db_datareader"));
+    }
+
+    [Fact]
+    public void TestUserHasPermissions()
+    {
+      var db = _fixture._agDatabase._listener.Primary.Database(_fixture._loginConfig.DefaultDatabase)._database;
+      var permissions = db.EnumDatabasePermissions(_fixture._loginConfig.LoginName);
+      Assert.NotNull(permissions.FirstOrDefault(p => p.PermissionType.Execute));
+    }

```

Comment:
These tests use `internal` fields. Accessing these `smo `objects could be moved to `TestAgDatabaseFixture`, or I could add accessors for roles and permissions to the `User` class instead. 

Either way I'll need to get `_agDatabase._listener.Primary` since user creation will be confined to the primary instance.